Formula/opencore-amr.rb

```diff
@@ -27,4 +27,17 @@ def install
     system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
     system "make", "install"
   end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      #include <opencore-amrwb/dec_if.h>
+      int main(void) {
+        void *s = D_IF_init();
```

Comment:
Hmm, why?

Reply:
`void*` can be literally anything, and there's no documentation in the header to suggest what you can do with what this returns. Ideally we should at least verify that the call to `D_IF_init` succeeded, but since we don't know what it spits out (at least not without more digging), we can't do that.

Reply:
Maybe we can just run `D_IF_exit` on the result immediately?

```suggestion
        void *s = D_IF_init();
        D_IF_exit(s);
```

Reply:
Yea, calling `D_IF_exit` is an improvement.